src/agent/onefuzz/src/monitor.rs

```diff
@@ -132,17 +132,16 @@ impl DirectoryMonitor {
                             }
                         }
                         CreateKind::Any | CreateKind::Other => {
-                            if self.report_directories {
```

Comment:
I'd prefer that we leave the short circuit in place here to avoid going to disk if we don't need to